client/gutenboarding/components/header/index.tsx

```diff
@@ -18,7 +18,7 @@ interface Props {
 }
 
 export default function Header( { isEditorSidebarOpened, toggleGeneralSidebar }: Props ) {
-	const { siteTitle, siteType } = useOnboardingState();
+	const { siteTitle, siteType } = useOnboardingSelect().getState();
```

Comment:
Will we able to roll this back? See https://github.com/Automattic/wp-calypso/pull/37118/files#r340602794

Reply:
I don't really think this is helpful, but I restored it to this diff. We can remove it later if we decide to.